src/components/LightUpChinatown/LightUpChinatownPage.tsx

```diff
@@ -81,7 +97,21 @@ const LightUpChinatownPage = () => {
           <br></br>
           {t('lightUpChinatown.ceremony2')}
         </FooterText>
+
+        <button
+          className={'button--filled'}
+          onClick={openModal}
+          value={ModalPaymentTypes.modalPages.light_up_chinatown}
+        >
+          Donation
+        </button>
       </BottomBanner>
+
+      <Modal
+        sellerId="light-up-chinatown"
+        sellerName="Light Up Chinatown"
+        costPerMeal={0}
```

Comment:
We should probably make `costPerMeal` optional in props to avoid having in the future when it's not relevant

Reply:
yeah i was playing around with it earlier, there needs to be another payment modal change dealing with nit changes like css, text copy and just cleanup in general.